projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -179,4 +179,10 @@ export class AuthService {
   setLogoutProgress(progress: boolean): void {
     (this.logoutInProgress$ as BehaviorSubject<boolean>).next(progress);
   }
+
+  afterRedirectFromCDCLogin(): void {
```

Comment:
let name the method without metioning CDC, because auth.service is a generic service, you can use oidc insetad